agent/bench-scripts/postprocess/uperf-prepare-xmlfile

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+# -*- mode: shell-script; indent-tabs-mode: t; sh-basic-offset: 8; sh-indentation: 8; sh-indent-for-case-alt: + -*-
+#echo "<profile name=\"protocol:\$protocol-test_type:\$test_type-wsize:\$wsize-threads:\$threads\">"
+
+echo "$@" >pre-script-args.txt
+test_type=stream
+while [ ! -z $1 ]; do
+	arg=`echo $1 | awk -F= '{print $1}'`
+	if [ $arg == "--test-type" ]; then
+		test_type=`echo $1 | awk -F= '{print $2}'`
+		break
+	fi
+	shift
+done
+	
+file=$pbench_tmp/uperf.xml
```

Comment:
Why hard code the file here instead of just printer to stdout and letting the caller redirect it to where it wants it?

Reply:
Why would we assume all controller pre-scripts' output be redirected to a file in a specific place?  The caller is pbench-run-benchmark-sample -if we want per-benchmark control over where this files gets redirected to, now we have to add even more configuration in the benchmark.json so pbench-run-benchmark-sample knows where to redirect this.  Then , what if a benchmark needs the pre-script to generate 2 files, or 3?